Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTreeView.cpp

```diff
@@ -491,6 +491,14 @@ namespace AzToolsFramework
             }
         }
 
+        void AssetBrowserTreeView::mouseDoubleClickEvent(QMouseEvent* ev)
+        {
+            if (ev->button() == Qt::MouseButton::LeftButton)
+            {
+                emit doubleClickedSignal(QModelIndex());
+            }
+        }
```

Comment:
you're neither calling accept() nor ignore on the event here. Do you want it to propagate further?
https://doc.qt.io/qt-6/qmouseevent.html

Reply:
Totally rewritten.